cmd/pg-schema-diff/version_cmd.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"runtime/debug"
+
+	"github.com/spf13/cobra"
+)
+
+func buildVersionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Get the version of pg-schema-diff",
+	}
+	cmd.RunE = func(_ *cobra.Command, _ []string) error {
+		buildInfo, ok := debug.ReadBuildInfo()
```

Comment:
Because we don't build the binary in our CI/CD pipeline, this is the best way to retrieve debug information for the binary.